cfgov/cfgov/settings/base.py

```diff
@@ -101,6 +101,18 @@
     {'import': 'ccdb5_ui', 'apps': ('ccdb5_ui', )},
 ]
 
+# These apps use "legacy" database routing
+# See: cfgov/v1/db_router.py
+LEGACY_APPS = [
+    'comparisontool',
+    'paying_for_college',
+    'retirement_api',
+    'knowledgebase',
+    'agreements',
+    'picard'
```

Comment:
Look carefully at the end of this line... missing comma! That is a tricky thing to catch.